app/src/main/java/com/example/keniphhomemaintenance/ui/navigation/NavArugmentParameters.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.keniphhomemaintenance.ui.navigation
+
+class NavArgParams {
+    companion object {
```

Comment:
How will this approach scale when we have 5 screens? 100? More? 

Ideally we create variables, including static variables, close to where we use them. So the ones related to Dwellings should live in some static context in a class that deals with Dwellings.

Reply:
Figured it could be grouped either with navigation stuff or dwellings stuff, but I like the idea of keeping it with dwellings. Renamed and relocated